openshift/scripts/ibm-ocp-template-uninstall.sh

```diff
@@ -88,7 +88,6 @@ if [[ -n "$TEMPLATE_FULL_UNINSTALL" ]]; then
     cd ..
     rm -rf ./$OM_TEMP_DIR
     check_exit_custom $GIT_CLONE_EXIT "Failed to download Operator Marketplace resource definitions. Ensure that you are connected to the Internet and can access GitHub."
-    check_exit_custom $OC_APPLY_EXIT "Failed to uninstall Operator Marketplace. Check the command output and try again."
```

Comment:
why do you remove this print statement?

Reply:
Seems to have prevented rerunning uninstall, e.g. if it fails partially, though not sure if it was due to the stuck deletion. I'll add it back in and see if it works when we test uninstall again.